overlay.nix

```diff
@@ -0,0 +1,3 @@
+final: prev:
+
+final.callPackage ./. { }
```

Comment:
This still use the locked one.

I took a glance of the impl of flake-compat. It exposes all flake outputs. So `(import ./.).overlays.default` should just work.

:thinking: Maybe dropping this file is also okay.

Reply:
I dropped this file, cause I don't know how test this.